Missing closing brace in testDeserializationRejectsOutOfRange causes test_session_config build failure

### Summary
`tests/unit/test_session_config.cpp` fails to compile because `testDeserializationRejectsOutOfRange()` is missing its closing brace, which cascades into parse errors on the two functions that follow and on the generated `test_session_config.moc`.

### Location
- **File(s):** `tests/unit/test_session_config.cpp`
- **Line(s) / function(s):** `TestSessionConfig::testDeserializationRejectsOutOfRange()` opens at L154 and ends at L229 without a closing brace, causing `testAllowInvalidCertificatesDefaultsOff()` (L230) and `testAllowInvalidCertificatesRoundTrip()` (L251) to be parsed as nested qualified-id declarations.

### Category
`functional`

### Severity
`high`

Justification: the default `make` build fails, which breaks CI and any contributor's first build from a clean checkout.

### Reproduction / Evidence
Run `make` from a clean checkout on `main`. The compiler emits:

```
tests/unit/test_session_config.cpp:230:64: error: qualified-id in declaration before '(' token
  230 | void TestSessionConfig::testAllowInvalidCertificatesDefaultsOff() {
tests/unit/test_session_config.cpp:251:62: error: qualified-id in declaration before '(' token
  251 | void TestSessionConfig::testAllowInvalidCertificatesRoundTrip() {
tests/unit/test_session_config.cpp:280:1: error: a function-definition is not allowed here before '{' token
  280 | QTEST_MAIN(TestSessionConfig)
build/test_session_config_autogen/include/test_session_config.moc:199:1: error: expected '}' at end of input
```

Verified by code analysis: `testDeserializationRejectsOutOfRange()` opens a function body at L154 and only has a single closing `}` at L229 which closes the final anonymous block; the outer function body is never closed before the next function declaration at L230.

### Expected Behavior
`make` succeeds and `test_session_config` builds.

### Actual Behavior
`make` fails with the errors above; `test_session_config` target is not built.

### Root Cause
`testDeserializationRejectsOutOfRange()` was added without a matching closing brace for the function body. The last `}` in the function closes the inner scope block rather than the function itself.

--- a/tests/unit/test_session_config.cpp
+++ b/tests/unit/test_session_config.cpp
@@ -227,6 +227,8 @@ void TestSessionConfig::testDeserializationRejectsOutOfRange() {
         QVERIFY(cfg.fromJson(j));
         QCOMPARE(cfg.codePage(), core::CodePage::ID::CP037);
     }
+}
+
 void TestSessionConfig::testAllowInvalidCertificatesDefaultsOff() {
     SessionConfig fresh;
     QVERIFY(!fresh.allowInvalidCertificates());

